src/Components/Views/Survey/survey.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import styled from 'react-emotion'
+import { Shades, Colors } from 'Elements/Colors/colors'
+import { StyledButton, buttonReset } from 'Elements/Buttons/button'
+import { PanelBase } from 'Elements/Foundations/foundation'
+
+const PanelWrapper = styled('div')`
+  ${PanelBase};
+  padding: 0.8rem;
+  text-align: center;
+`
+const CancelSurvey = styled('button')`
+  ${buttonReset};
+  font-size: 0.6rem;
+  color: ${Shades.overcast};
+  padding-top: 0.6rem;
+  &:hover {
+    color: ${Colors.teal};
+  }
+`
+
+const FieldSet = styled('fieldset')`
+  all: unset;
+`
+const FieldTitle = styled('legend')`
+  all: unset;
+  font-size: 1rem;
+  padding: 0.2rem 3.5rem 1rem;
+  font-weight: 600;
+`
+const FieldOption = styled('button')`
+  ${StyledButton};
+  width: 100%;
+  margin-bottom: 4px;
+  font-size: 0.875rem;
+`
+
+function FieldOptions({ options, name }) {
+  return options.map(option => {
+    const { text, value } = option
+    return (
+      <FieldOption onClick={() => console.log(`${name}:${value}`)}>
```

Comment:
a unique key is required, there is no unique property in the dataset so using index* (don't forsee any option being dynamically updated)

<img width="533" alt="screen shot 2018-07-17 at 10 10 52 am" src="https://user-images.githubusercontent.com/1546207/42833787-0d58b3f0-89aa-11e8-8f15-18f6270a6bf9.png">